seonoh/ArchitectureStudy/app/src/main/java/com/example/seonoh/seonohapp/CoinAdapter.kt

```diff
@@ -0,0 +1,29 @@
+package com.example.seonoh.seonohapp
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.example.seonoh.seonohapp.model.UseCoinModel
+
+class CoinAdapter : RecyclerView.Adapter<CoinItemViewHolder>() {
+
+    private val mData: ArrayList<UseCoinModel> by lazy { ArrayList<UseCoinModel>() }
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): CoinItemViewHolder {
+
+        val mView = LayoutInflater.from(parent.context).inflate(R.layout.coin_item, parent, false)
+        return CoinItemViewHolder(mView)
+    }
+
+    override fun getItemCount(): Int = mData!!.size
+
+    fun addCoinData(data: ArrayList<UseCoinModel>) {
+        mData!!.addAll(data)
+        notifyDataSetChanged()
+    }
+
+    override fun onBindViewHolder(holder: CoinItemViewHolder, position: Int) {
+        holder.bind(mData!![position])
```

Comment:
notnull 단언자(!!) 를 제거 할 수 있습니다
```suggestion
        holder.bind(mData[position])
```

Reply:
네 감사합니다 반영했습니다!

Reply:
@seonoh 
반영 커밋 남겨주세요

Reply:
네 감사합니다. 커밋남깁니다.
7896ebf